{"name": "test_login_fail[fail_data3]", "status": "passed", "description": "\n        用例描述：测试登录失败用例\n        ", "attachments": [{"name": "log", "source": "02604067-a729-48ce-9919-a9439b46908d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "c34dd56c-b8b0-4da5-9320-4a5370fdb0ab-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "fail_data", "value": "{'sUserName': '16533907260', 'sPassword': ''}"}], "start": 1587118962538, "stop": 1587118963209, "uuid": "78b7a2d2-9143-49d6-9ba7-c8d8889ae901", "historyId": "db57e921a10ef9659acaa552a08558fa", "testCaseId": "8000cb86548fa2cebd0d986e95a5b795", "fullName": "test_case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "登录失败用例"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "11288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}